package/network/services/uhttpd/files/uhttpd.init

```diff
@@ -220,6 +220,9 @@ start_instance()
 		[ -s "$file" ] && procd_append_param command -H "$file"
 	done
 
+	config_get_bool mdns "$cfg" mdns 1
+	[ "$mdns" -ne 0 ] && procd_add_mdns "http" "tcp" "80" "daemon=uhttpd"
```

Comment:
Please do not hard code port 80.
Use the ports from `http` and `https`

Reply:
Given the limit of 1 `umdns` advertisement per `procd` service my latest commit matches one port from `listen_http` and brings up an advertisement on it.